packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,45 @@
+# 50.13.0 - January 13, 2021
+
+## Added
+
+#### salesforcedx-vscode-apex
+
+- Added debug button to test sidebar to automatically run Apex Replay Debugger after test run ([PR #2804](https://github.com/forcedotcom/salesforcedx-vscode/pull/2804))
+
+#### salesforcedx-vscode-core
+
+- Run Apex tests using the Apex library ([PR #2828](https://github.com/forcedotcom/salesforcedx-vscode/pull/2828))
+
+#### salesforcedx-vscode-soql
```

Comment:
Just posing the question in case this has changed - are we now including SOQL in the changelog since it's part of the publish process?